Mailer/README.md

```diff
@@ -535,12 +641,15 @@ for that, please visit `/list/new` to create newsletter via web admin.
 
 ```json5
 {
-	"email": "admin@example.com", // String; email of the user
-	"user_id": 123, // Integer; ID of the user
+  "email": "admin@example.com", // String; email of the user
+  "user_id": 123, // Integer; ID of the user
 	
-	// one of the following is required
-	"list_id": 14, // Integer; ID of the newsletter list you're subscribing the user to
-	"list_code": "alerts", // String; code of the newsletter list you're subscribing the user to
+  // one of the following is required
+  "list_id": 14, // Integer; ID of the newsletter list you're subscribing the user to
+  "list_code": "alerts", // String; code of the newsletter list you're subscribing the user to
+
+  // optional 
+  "variant_id": 1, // Integer;  ID of newsletter variant to subscribe
```

Comment:
Please use `variant_code` instead of ID. It's prefered option in cases where only one of those two is available.

Reply:
It already was in the previous implementation:
https://github.com/remp2020/remp/blob/a56acdceabfb2a88581ae056f7d89d7dfab75658/Mailer/app/api/v1/Handlers/Users/SubscribeHandler.php#L151
It just wasn't for some reason in the documentation. However, it can be used by some clients.

Reply:
Sigh. In that case let's keep it as is. Thanks for clarifying.